pyproject.toml

```diff
@@ -24,7 +24,7 @@ dependencies = [
   'pooch',
   'scooby>=0.5.1',
   'typing-extensions',
-  'vtk<9.4.0',
+  'vtk<9.5.0',
```

Comment:
Assuming there are no surprises, I think we should add tests for Python 9.13 and add to classifiers in pyproject.toml here.  vtk 9.4.x is the thing holding back support for this version.